ocs_ci/ocs/ui/base_ui.py

```diff
@@ -445,6 +452,9 @@ def navigate_installed_operators_page(self):
             self.page_nav["installed_operators_page"], enable_screenshot=False
         )
         self.page_has_loaded(retries=25, sleep_time=10)
+        if self.ocp_version_full >= version.VERSION_4_9:
+            self.do_click(self.page_nav["drop_down_projects"])
```

Comment:
Please check the toggle button status here, otherwise test will fail in some cases

Reply:
I dont need to check toggle button status because I navigate to installed_operators_page at the beginning of this method.

Reply:
I disagree, and we can discuss this in today's meeting